spec/requests/registrations_spec.rb

```diff
@@ -75,6 +75,23 @@
 
         expect(response.body).to include("View more sign in options")
       end
+
+      it "creates a user with a random profile image if none was uploaded" do
+        name = "test"
+        # rubocop:disable RSpec/AnyInstance
+        allow_any_instance_of(ProfileImageUploader).to receive(:download!)
```

Comment:
Not ideal to use `allow_any_instance_of` I think... Taking suggestions here 😅

Reply:
I think it's ok and wouldn't worry too much.